Added basic auth quick start guide (#93)
quantori/Behavioral.Automation: BINDINGS.md

[When("(.*?) element among (.*) (is| is not) expanded")]
[Then("(.*?) element among (.*) should (be|be not) expanded")]
Example:
When first element among "Search results" grid rows is expanded

# Scenario hooks guides
## Basic auth hook

> In order to make the following should be done:

Reviewer: ```suggestion
In order to make basic authentication work user should do the following:
```

Author: Changed

* @BasicAuth tag should be added to scenario which requires authentication
* AutomationConfig.json should have credentials for basic auth. Please refer to [example](Behavioral.Automation.DemoBindings/AutomationConfig.json).
* If you want to use credentials from different sources IBasicAuthConfig interface can be used

> * If basic auth should be ignored on env then IgnoreAuth field can be used to disable basic auth

Reviewer: ```suggestion
* To ignore basic auth on given env the IgnoreAuth field can be used to disable basic auth
```

Author: Changed
